.gitignore

```diff
@@ -30,3 +30,8 @@ bin
 /sormas-cargoserver/.env
 /sormas-cargoserver/custom.env
 /sormas-cargoserver/custom.properties
+
+# see #4469
```

Comment:
The comment referencing the issue is obsolete, the info is available via git annotations anyway.

Reply:
true story :D